internal/gitinterface/changes.go

```diff
@@ -3,13 +3,34 @@
 package gitinterface
 
 import (
+	"container/heap"
 	"fmt"
 	"sort"
 
 	"github.com/gittuf/gittuf/internal/third_party/go-git"
 	"github.com/gittuf/gittuf/internal/third_party/go-git/plumbing/object"
 )
 
+type diffHeap []string
```

Comment:
Can you move this to the end of the file? We try to have helpers at the end or in a helpers.go file in the package...

I'll also double check the licensing around this, as I think it's based off the standard heap example in the docs.

Reply:
I moved it to the end of the file